grails-app/services/org/pih/warehouse/importer/LocationImportDataService.groovy

```diff
@@ -7,32 +7,22 @@
  * the terms of this license.
  * You must not remove this notice, or any other, from this software.
  **/
-package org.pih.warehouse.data
+package org.pih.warehouse.importer
 
 import grails.gorm.transactions.Transactional
 import grails.validation.ValidationException
-import org.pih.warehouse.core.Address
+import org.pih.warehouse.LocalizationUtil
 import org.pih.warehouse.core.Location
 import org.pih.warehouse.core.LocationGroup
+import org.pih.warehouse.core.LocationService
 import org.pih.warehouse.core.LocationType
-import org.pih.warehouse.core.LocationTypeCode
 import org.pih.warehouse.core.Organization
-import org.pih.warehouse.importer.ImportDataCommand
-import org.pih.warehouse.inventory.Inventory
-import org.pih.warehouse.LocalizationUtil
-
-import javax.annotation.Nullable
 
 @Transactional
-class LocationDataService {
-
-    def organizationService
-    def identifierService
+class LocationImportDataService implements ImportDataService {
```

Comment:
I know there was a reason it had not been named as `ImportDataService` - since it's also been used for exports and the Import name did not fit well.

Reply:
Yeah let's talk about that on the tech huddle. ImportDataService is a little awkward.

Reply:
Nothing to do now, but after more thought I'm thinking we're probably going to call these classes *DataImportService or *DataMigrationService. If we combine data import and data export into a single implementation class then it'll probably be the latter with the potential for using interfaces for the two types of data migration activities.
```
class LocationDataMigrationService implements DataImportService, DataExportService {
    void validateData(ImportDataCommand command) { ... } 
    void importData(ImportDataCommand command) { ... }
    List exportData(ExportDataCommand command) { ... }
}
```
Not entirely sure yet if the command classes will follow suit or if I am ok with command classes named by an action (import data) rather than a noun (data import).